Calibration/HcalCalibAlgos/test/gammaJetAnalysis_cfg.py

```diff
@@ -1,13 +1,66 @@
 import FWCore.ParameterSet.Config as cms
 process = cms.Process('ANALYSIS')
 
-process.load('FWCore.MessageService.MessageLogger_cfi')
 process.load('Configuration.StandardSequences.Services_cff')
 process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
```

Comment:
@bsunanda 
please change this to `process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff')` in order to consume conditions from the new backend. The old one is decommissioned.